### PR TITLE
Feature/file system

### PR DESCRIPTION
**Pull request by MartinsRui**

### CODE REVIEW

File: src/IO/FileSystem/DotNetFileSystemProvider.cs [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+namespace RapidCore.IO.FileSystem
+{
+    public class DotNetFileSystemProvider : IFileSystemProvider
+    {
+        public FileStream OpenReadFile(string path)
+        {
+            return File.OpenRead(path);
+        }
+
+        Stream IFileSystemProvider.OpenReadFile(string path)
```
Comment (nover) [inline src/IO/FileSystem/DotNetFileSystemProvider.cs]: Is there need for an explicit interface reference here?

File: src/IO/FileSystem/DotNetFileSystemProvider.cs [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+        /// </summary>
+        /// <returns>
+        /// the current working directory
+        /// </returns>
+        public string Cwd()
+        {
+            return Directory.GetCurrentDirectory();
+        }
+
+        public void Move(string sourceFile, string destFile)
```
Comment (nover) [inline src/IO/FileSystem/DotNetFileSystemProvider.cs]: No documentation for Move function

File: src/rapidcore.csproj [hunk 3]
```diff
@@ -27,4 +27,8 @@
     <None Remove="../stylecop.json" />
     <AdditionalFiles Include="../stylecop.json" />
   </ItemGroup>
+
+  <ItemGroup>
```
Comment (nover) [inline src/rapidcore.csproj]: Is this reference required when we move to netstandard 1.6?

**Review by nover — CHANGES_REQUESTED** [review]

File: src/rapidcore.csproj (hunk 3 above)
Comment (MartinsRui) [reply src/rapidcore.csproj]: not required anymore.
fixed.

File: src/IO/FileSystem/DotNetFileSystemProvider.cs (hunk 2 above)
Comment (MartinsRui) [reply src/IO/FileSystem/DotNetFileSystemProvider.cs]: fixed

File: src/IO/FileSystem/DotNetFileSystemProvider.cs (hunk 1 above)
Comment (MartinsRui) [reply src/IO/FileSystem/DotNetFileSystemProvider.cs]: fixed

**Review by nover — APPROVED** [review]

**Pull request merged**